zanata-war/src/main/java/org/zanata/ApplicationConfiguration.java

```diff
@@ -76,6 +80,9 @@
     @Getter
     private static final int defaultMaxFilesPerUpload = 100;
 
+    @Getter
+    private static final int defaultAnonymousSessionTimeoutMinutes = 30;
```

Comment:
Not to be picky (surely not!) but why minutes here and calculated to seconds later?